src/components/SigninPopup/SigninPopup.js

```diff
@@ -14,42 +15,70 @@ function SigninPopup({ ...props }) {
     usePopups().popups.signInPopup;
   const { openSignUpPopup } = usePopups().popups.signUpPopup;
 
+  const { onLogin, authServerError, changeAuthError } = useAuth();
+
+  function handleClose() {
+    closeSignInPopup();
+    changeAuthError('');
+    resetForm();
+  }
+
+  function handleSubmit() {
+    const userData = { email, password };
+    onLogin(userData);
+    resetForm();
+  }
+
   return (
     <>
```

Comment:
* `<>` empty tags are not needed if you have a common tag that wraps the whole markup of the component